src/entities/hr/hr.entity.ts

```diff
@@ -1,17 +1,21 @@
-import { Entity, PrimaryGeneratedColumn, Column, BaseEntity, PrimaryColumn } from 'typeorm';
+import { Entity, Column, BaseEntity, PrimaryColumn, OneToOne, JoinColumn } from 'typeorm';
 import { hrInterface } from '../types/hr';
+import { User } from '../User/User.entity';
 
 @Entity('hr')
 export class Hr extends BaseEntity implements hrInterface {
   @PrimaryColumn()
   id: string;
+  @OneToOne(() => User, { 'cascade': true })
```

Comment:
enter between id and OneToOne. In that case we have Primary colum which isn't generated automaticaly and user column which store user Id. So maybe You should delete id: column and put 'primary': true to OneToOne

Reply:
Robiłem testy i przy tym zapisie jest ok.